lib/osf-components/addon/components/search-page/filter-facet/component.ts

```diff
@@ -48,12 +48,20 @@ export default class FilterFacet extends Component<FilterFacetArgs> {
     @tracked hasMoreValueOptions = false;
     @tracked nextPageCursor = '';
 
+    get shouldShowTopValues() {
+        return !this.args.property.propertyPathKey.includes('creator');
```

Comment:
Would this break for preprint's institution affiliation filter?

Reply:
oh, i forgot about that one... yeah i'll make it check `!== 'creator'`, is fine